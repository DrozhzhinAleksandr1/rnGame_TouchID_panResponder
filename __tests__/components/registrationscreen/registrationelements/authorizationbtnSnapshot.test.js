import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { AuthorizationBtn } from '../../../../app/components/registrationscreen/registrationelements/authorizationbtn';

it('create snapshot AuthorizationBtn', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<AuthorizationBtn />);
    expect(toolbarSnapshot).toMatchSnapshot();
});