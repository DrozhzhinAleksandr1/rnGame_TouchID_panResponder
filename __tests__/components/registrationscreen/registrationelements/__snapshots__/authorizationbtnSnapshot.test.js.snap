// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot AuthorizationBtn 1`] = `
<Component
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "marginTop": 20,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#ffffff",
        "fontFamily": "Roboto-Regular",
        "fontSize": 16,
        "lineHeight": 19,
      }
    }
  />
  <TouchableWithoutFeedback
    accessibilityLabel="BtnAuthorizationInRegistrationID"
    onPress={[Function]}
    testID="BtnAuthorizationInRegistrationID"
  >
    <Component>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#4BCEA9",
            "fontFamily": "Roboto-Regular",
            "fontSize": 18,
            "fontWeight": "bold",
            "lineHeight": 19,
            "marginLeft": 5,
          }
        }
      />
    </Component>
  </TouchableWithoutFeedback>
</Component>
`;
